requirements.txt

```diff
@@ -173,3 +173,7 @@ ndg-httpsclient==0.3.2
 # sha256: bTs1EQn0GQzZdA12M4RpkIGl8uuNc9qn5IUAn7ZGGMA
 # sha256: 5PgdU8Uz9r2VJrBH8Ef3sQHCSrFzOcGnrY-YslwQHqs
 pyasn1==0.1.7
+# sha256: 02C9MYmPnfWfb6p4ZVEGW7pFs15-48ObOBtPv-9zkvQ
+South==1.0.2
+# sha256: NmfSakH-wwNkoO9yWAgyylMogC1VP21ucq9awhyzY2U
+django-nose==1.3
```

Comment:
This is making ```pip``` complain and not install packages. The old one is a github URL, we should remove that in favor of using the pypi version anyhow.